src/View/PasswordQuickInput.ts

```diff
@@ -16,8 +16,20 @@
 
 import * as vscode from 'vscode';
 
-// NOTE ASCII characters have codes ranging from u+0000 to u+007f
+// NOTE
+//
+//   u+0000-u+001f: control ascii codes,    ascii (ALLOWED)
+//   u+0020-u+007f: printable ascii codes,  ascii (ALLOWED)
+//   u+0080-u+00ff: extended ascii codes,   extended ascii (NOT ALLOWED)
+//
+// By experience, Linux os takes all extended ascii characters(u+0000-u+00ff) for passwords, even
+// controll asciis. whereas 'eslint' recommends not using regex including control ascii codes.
+// Therefore, we allow users to put control ascii codes.
+//
+// By experience, the passwords including extended ascii characters are not working when loging in
+// to remote vscode. Therefore, here we allow only 'ascii characters'(0x00-0x7f).
```

Comment:
@jyoungyun I tried several tests under what you've mentioned, I summarized here what I've found out. It seems that leaving the codes as-is is enough for now. PTAL :-D